<<<<<<< HEAD
<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FreqsetsWidget</class>
 <widget class="QWidget" name="FreqsetsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1242</width>
    <height>566</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FreqsetsWidget</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>1210</x>
     <y>450</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string> MHz</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Add">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>510</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>添加</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Freq">
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>450</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Delete">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>480</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget_Freqset">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>0</y>
     <width>171</width>
     <height>441</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>450</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string> 频率</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget_AllFreq">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>361</y>
     <width>961</width>
     <height>201</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_InTable">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>540</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>入表</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
=======
<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FreqsetsWidget</class>
 <widget class="QWidget" name="FreqsetsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1242</width>
    <height>566</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FreqsetsWidget</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>1210</x>
     <y>450</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string> MHz</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Add">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>510</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>添加</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Freq">
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>450</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Delete">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>480</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget_Freqset">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>0</y>
     <width>171</width>
     <height>441</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>450</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string> 频率</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget_AllFreq">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>361</y>
     <width>961</width>
     <height>201</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_InTable">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>540</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>入表</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
>>>>>>> origin/master
